tldw_Server_API/tests/Sharing/test_workspace_deletion_hook.py

```
@@ -0,0 +1,54 @@
+@pytest.mark.asyncio
+async def test_on_workspace_deleted_revokes_shares(repo):
+    # Create a share and a token
+    await repo.create_share(
+        workspace_id="ws-del",
+        owner_user_id=1,
+        share_scope_type="team",
+        share_scope_id=10,
+        created_by=1,
+    )
+    await repo.create_token(
+        token_hash="delhash",
```

Ruff (0.15.6)

[error] Possible hardcoded password assigned to argument: "token_hash"

(S106)